src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { useCallback, useMemo } from 'react';
+import { deleteTodos } from '../../api/todos';
+import { Error } from '../../types/ErrorType';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setFilterType: (filter: FilterType) => void;
+  filterType: FilterType;
+  todos: Todo[];
+  completeTodos: Todo[];
+  setSelectedTodosIds: React.Dispatch<React.SetStateAction<number[]>>;
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setErrorNotification: React.Dispatch<React.SetStateAction<string | null>>;
+  errorNotification: string | null;
+  selectedTodosIds: number[];
+};
+
+export const Footer: React.FC<Props> = ({
```

Comment:
Reread [Task checklist point 3](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md). Component names examples there more meaningful.

`Header` here also not a good name for the component.